src/components/missionsComponent/MissionItem.js

```diff
@@ -0,0 +1,53 @@
+// import React from 'react';
+// import PropTypes from 'prop-types';
+// import { useDispatch } from 'react-redux';
+// import { joinMission, leaveMission } from '../../redux/missions/missions';
+
+// const MissionList = ({
+//   id, name, description, reserved,
+// }) => {
+//   const dispatch = useDispatch();
+//   return (
+//     <tr className="row">
+//       <td className="column name">{name}</td>
+//       <td className="column description">{description}</td>
+//       <td className="column status_column">
+//         {reserved
+//         && <p className="ActiveStatus">Active Member</p>}
+//         {!reserved
+//         && <p className="status">NOT A MEMBER</p>}
+//       </td>
+//       <td className="column column_button">
+//         {reserved
```

Comment:
- [Optional] Please remove all commented codes from your work. This is to keep your work clean.